CLAUDE.md

```diff
@@ -0,0 +1,269 @@
+# CLAUDE.md
+
+This file provides guidance to Claude Code (claude.ai/code) when working with code in this repository.
+
+## Development Commands
+
+### Building and Development
+- `npm run lint` - Run ESLint and TypeScript checks (max 0 warnings)
```

Comment:
I realized `lint` might be broken these days because we don't have a single tsconfig.json. Probably better to remove it from here since there is a dedicated typechecking action that covers it all.